app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -67,13 +67,17 @@ def prepare_saml_logout_response
     @saml_response = OneLogin::RubySaml::Logoutresponse.new(params[:SAMLResponse])
   end
 
+  def slo_session
```

Comment:
the `slo` acronym is used pretty widely in the app. `grep -i slo config | wc` suggests at least 15 hits.